### ISSUE TITLE
log spam: "named cookie not present" can appear frequently

### ISSUE THREAD

**Issue by programmerq** [gravitational/teleport#10559]
### Description

**What happened**:

tons of messages like the following can appear in the logs of the teleport proxy component:

```
2022-02-14T16:22:28Z WARN [WEB]       http: named cookie not present request:GET /web/login web/apiserver.go:2526
```

There's really not much action a teleport administrator can take on these messages, so it ends up being unnecessary noise at the WARN level.

**What you expected to happen**:

This type of log should be a DEBUG level or absent altogether.

### Reproduction Steps

Attempt to access a teleport cluster with an expired session that wasn't properly torn down on the browser side.

### Server Details
- Teleport version (run `teleport version`): 
- Server OS (e.g. from `/etc/os-release`):
- Where are you running Teleport? (e.g. AWS, GCP, Dedicated Hardware):
- Additional details:

### Client Details
- Tsh version (`tsh version`):
- Computer OS (e.g. Linux, macOS, Windows):
- Browser version (for UI-related issues):
- Installed via (e.g. apt, yum, brew, website download):
- Additional details:

### Debug Logs
Please include or attach debug logs, when appropriate. Obfuscate sensitive information!
- Start Teleport with --debug flag (`teleport --debug`)
- Run tsh with --debug flag (`tsh --debug`)

cc @Aharic 

gz#4077

**Comment by stevenGravy** [comment]
happens a lot due to health checks

**Comment by Aharic** [comment]
Have a few customers asking if we can either suppress these (wrap them up if they're happening in a short sequence into a single message with a counter) or elevate them so they don't appear when customers are running INFO severity.

**Comment by zmb3** [comment]
Fixed in #20694
